Kite SDK/PSPrintSDK/OLKitePrintSDK.m

```diff
@@ -232,10 +232,18 @@ + (NSString *)instagramClientID{
 }
 #endif
 
-+ (void)fetchRemotePlist{
++ (void)fetchRemotePlistsWithCompletionHandler:(void(^)())handler{
+    [OLKitePrintSDK fetchRemotePlistWithURL:nil completionHandler:^(NSError *error){
+        [OLKitePrintSDK fetchRemotePlistWithURL:[NSString stringWithFormat:@"https://sdk-static.s3.amazonaws.com/kite-ios-remote-%@.plist", [OLKitePrintSDK apiKey]] completionHandler:^(NSError *error2){
+            handler();
```

Comment:
Any chance if that on error we can fall back to using the global plist file https://sdk-static.s3.amazonaws.com/kite-ios-remote.plist. This way we only need to put in specific plist files for the high profile clients. Longer term we can maybe dynamically generate these files but for now I know I'll be a bottleneck making them for everyone if we don't have a fallback